src/styles/blocks/card.scss

```
@@ -0,0 +1,82 @@
+.card {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    color: $text-color-main-accent;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    font-size: 10px;
+    line-height: 14px;
+    color: $text-color-secondary;
+  }
+
+  &__review-box {
+    margin-bottom: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+  }
+
+  &__review {
+    font-size: 10px;
+    line-height: 14px;
+    color: $text-color-main-accent;
+  }
+
+  &__price {
+    margin-bottom: 16px;
+    display: flex;
+    justify-content: space-between;
+
+    &-title {
+      font-size: 12px;
+      line-height: 18px;
+      color: $text-color-secondary;
+    }
+
+    &-value {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+    }
+  }
+
+  &__buy-button {
+    box-sizing: border-box;
+    height: 40px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: $color-blue-accent;
+    color: $color-white;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    border-radius: 5px;
+    border: 1px solid $color-blue-accent;
+    text-decoration: none;
+    text-transform: uppercase;
+  }
+
+  &__buy-button:hover {
+    color: $color-blue-accent;
+    background-color: $color-white;
+  }
```

use the power of nesting
```suggestion

    &:hover {
      color: $color-blue-accent;
      background-color: $color-white;
    }
  }
```